src/utils/composeProviders.tsx

```diff
@@ -1,7 +1,7 @@
-import {createElement, FC, ReactElement} from 'react';
+import {FC, ReactElement, createElement} from 'react';
 
 class ProviderCompositor {
-  public providers: {provider: FC<any>, props: any}[] = [];
+  public providers: {provider: FC<any>; props: unknown}[] = [];
```

Comment:
Do we use `;` for inline object type declarations, not `,`?

Reply:
Prettier prettifies in this way